Node Creation Failure

How do you fix this:

```bash
$ EKS_CLUSTER_NAME='dk-cluster-1'
$ eksctl create cluster \  
 --cluster-name ${EKS_CLUSTER_NAME} \
 --kubeconfig ~/.kube/config \
 --nodes 3 \
 --ssh-public-key ~/.ssh/${EKS_CLUSTER_NAME}.id_rsa.pub
2018-06-13T10:05:48-07:00 [ℹ]  importing SSH public key "/Users/jmenchaca/.ssh/dk-cluster-1.id_rsa.pub" as "EKS-dk-cluster-1"
2018-06-13T10:05:49-07:00 [ℹ]  creating EKS cluster "dk-cluster-1"
2018-06-13T10:05:49-07:00 [ℹ]  creating VPC stack "EKS-dk-cluster-1-VPC"
2018-06-13T10:05:49-07:00 [ℹ]  creating ServiceRole stack "EKS-dk-cluster-1-ServiceRole"
2018-06-13T10:06:29-07:00 [✔]  created ServiceRole stack "EKS-dk-cluster-1-ServiceRole"
2018-06-13T10:07:20-07:00 [✔]  created VPC stack "EKS-dk-cluster-1-VPC"
2018-06-13T10:07:20-07:00 [ℹ]  creating control plane "dk-cluster-1"
2018-06-13T10:17:41-07:00 [✔]  created control plane "dk-cluster-1"
2018-06-13T10:17:41-07:00 [ℹ]  creating DefaultNodeGroup stack "EKS-dk-cluster-1-DefaultNodeGroup"
2018-06-13T10:21:22-07:00 [✔]  created DefaultNodeGroup stack "EKS-dk-cluster-1-DefaultNodeGroup"
2018-06-13T10:21:22-07:00 [✔]  all EKS cluster "dk-cluster-1" resources has been created
2018-06-13T10:21:22-07:00 [ℹ]  wrote "/Users/jmenchaca/.kube/config"
2018-06-13T10:21:27-07:00 [ℹ]  the cluster has 0 nodes
2018-06-13T10:21:27-07:00 [ℹ]  waiting for at least 3 nodes to become ready
panic: interface conversion: runtime.Object is nil, not *v1.Node

goroutine 1 [running]:
github.com/weaveworks/eksctl/pkg/eks.(*ClusterConfig).WaitForNodes(0xc4202f3d40, 0xc42057a5a0, 0x0, 0x0)
	/go/src/github.com/weaveworks/eksctl/pkg/eks/nodegroup.go:103 +0x7fa
main.doCreateCluster(0xc4202f3d40, 0xc420171cb0, 0xc420171c80)
	/go/src/github.com/weaveworks/eksctl/cmd/eksctl/create.go:153 +0x485
main.createClusterCmd.func1(0xc4201efb80, 0xc42033c700, 0x0, 0x8)
	/go/src/github.com/weaveworks/eksctl/cmd/eksctl/create.go:57 +0x2a
github.com/weaveworks/eksctl/vendor/github.com/spf13/cobra.(*Command).execute(0xc4201efb80, 0xc42033c680, 0x8, 0x8, 0xc4201efb80, 0xc42033c680)
	/go/src/github.com/weaveworks/eksctl/vendor/github.com/spf13/cobra/command.go:766 +0x2c1
github.com/weaveworks/eksctl/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0x2a1cc00, 0xc420171f68, 0x1ca8d86, 0xc420171f78)
	/go/src/github.com/weaveworks/eksctl/vendor/github.com/spf13/cobra/command.go:852 +0x30a
github.com/weaveworks/eksctl/vendor/github.com/spf13/cobra.(*Command).Execute(0x2a1cc00, 0x1ff01a3, 0x1)
	/go/src/github.com/weaveworks/eksctl/vendor/github.com/spf13/cobra/command.go:800 +0x2b
main.main()
	/go/src/github.com/weaveworks/eksctl/cmd/eksctl/main.go:28 +0x2d

```

--- a/pkg/eks/nodegroup.go
+++ b/pkg/eks/nodegroup.go
@@ -99,14 +99,15 @@ func (c *ClusterConfig) WaitForNodes(clientSet *clientset.Clientset) error {
 		select {
 		case event, _ := <-watcher.ResultChan():
 			logger.Debug("event = %#v", event)
-			if event.Type != watch.Deleted {
-				node := event.Object.(*corev1.Node)
-				if isNodeReady(node) {
-					counter++
-					logger.Debug("node %q is ready", node.ObjectMeta.Name)
-				} else {
-					logger.Debug("node %q seen, but not ready yet", node.ObjectMeta.Name)
-					logger.Debug("node = %#v", *node)
+			if event.Object != nil && event.Type != watch.Deleted {
+				if node, ok := event.Object.(*corev1.Node); ok {
+					if isNodeReady(node) {
+						counter++
+						logger.Debug("node %q is ready", node.ObjectMeta.Name)
+					} else {
+						logger.Debug("node %q seen, but not ready yet", node.ObjectMeta.Name)
+						logger.Debug("node = %#v", *node)
+					}
 				}
 			}
 		case <-timer:

